TROUBLESHOOTING.md

```diff
@@ -1,7 +1,70 @@
-# Performance Caveats
+# Debugging
+
+Note that the information in this section is subject to be removed in future releases of
+the _PyTorch/XLA_ software, since many of them are peculiar to a given internal implementation
+which might change.
+
+Sometimes, bad things happen and a deeper look into the _PyTorch/XLA_ stack is necessary.
```

Comment:
I'm not sure about "bad things happen" - it's too ambiguous. "To diagnose issues, we can use the execution metrics and counters provided by _PyTorch/XLA_" covers both this and the following phrase and it sounds a bit better to me.